src/memory.h

```diff
@@ -303,8 +303,9 @@ class dcu_c
   int m_line_size; /**< cache line size */
   int m_banks; /**< number of cache banks */
   int m_latency; /**< cache access latency */
-  bool m_ptx_sim; /**< gpu cache */
+  bool m_acc_sim; /**< gpu cache */
   bool m_igpu_sim; /**< intel gpu cache */
+  bool m_ptx_sim; /**< gpu cache */
```

Comment:
aren't we replacing m_ptx_sim with m_acc_sim? then delete this?
or if m_acc_sim is equal to (m_ptx_sim || m_igpu_sim) then, can we change line 1492 or 1524 to if (m_acc_sim), not if (m_acc_sim || m_igpu_sim)?

Reply:
m_acc_sim = (m_ptx_sim || m_igpu_sim)